keps/sig-node/2371-cri-pod-container-stats/README.md

```diff
@@ -103,82 +103,82 @@ Summary API has two interfaces:
+|                              |WorkingSetBytes       |container_memory_working_set_bytes              |Pod and Container               |CRI                   |CRI                        |  memory.usage_in_bytes (extra if logic) | memory.usage_in_bytes (extra if logic)
+|                              |RSSBytes              |container_memory_rss                            |Pod and Container               |cAdvisor              |CRI                        | (memory.stat) total_rss    |  (memory.stat) anon
+|                              |PageFaults            |N/A                                             |Pod and Container               |cAdvisor              |CRI                        | (memory.stat) pgfault      |  (memory.stat) pgfault 
+|                              |MajorPageFaults       |N/A                                             |Pod and Container               |cAdvisor              |CRI                        | (memory.stat) pgmajfault   |  (memory.stat) pgmajfault
+|                              |N/A                   |container_memory_cache                          |N/A                             |cAdvisor              |CRI or N/A                 | (memory.stat) cache        |  (memory.stat) file
+|                              |N/A                   |container_memory_failcnt                        |N/A                             |cAdvisor              |CRI or N/A                 |  memory.failcnt                N/A
+|                              |N/A                   |container_memory_failures_total                 |N/A                             |cAdvisor              |CRI or N/A                 | (memory.stat) pg_fault && pg_maj_fault |
+|                              |N/A                   |container_memory_mapped_file                    |N/A                             |cAdvisor              |CRI or N/A                 | (memory.stat) mapped_file  |  (memory.stat) file_mapped
+|                              |N/A                   |container_memory_max_usage_bytes                |N/A                             |cAdvisor              |CRI or N/A                 | memory.max_usage_in_bytes  |  memory.max
+|                              |N/A                   |container_memory_swap                           |N/A                             |cAdvisor              |CRI or N/A                 | (memory.stat) swap  |  memory.swap.current - memory.current
+|ProcessStats                  |ProcessCount          |container_processes                             |Pod                             |cAdvisor              |CRI                        |  Process
+|AcceleratorStats              |Make                  |N/A (too lazy to find the mapping)              |Container                       |cAdvisor              |cAdvisor or N/A            |  accelerators/nvidia.go    | accelerators/nvidia.go
```

Comment:
AFAIK we still plan on dropping these. do we need to include them in this change?